pkg/minishift/provisioner/minishift_provisioner.go

```diff
@@ -111,8 +111,12 @@ func (provisioner *MinishiftProvisioner) Provision(swarmOptions swarm.Options, a
 	}
 	provisioner.EngineOptions.StorageDriver = storageDriver
 
+	print("-- Setting hostname ... ")
```

Comment:
It will even result in a writing to `stderr`; missed during the integration fix.